src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    add(num) {
+      obj.result += num;
```

Comment:
Use `this` instead of `obj`

Reply:
Hi) thanks for the comments, I hope I understood everything correctly and corrected it)